RHEcosystemAppEng/dbaas-operator
# DBaaS operator dependencies installation

## Description


The PR will install dependencies/components along with DBaaS:

MongoDB Atlas Operator
Crunchy Bridge Operator
dbaas-dynamic-plugin
console-telemetry-plugin
<img width="1460" alt="Screen Shot 2021-08-31 at 3 12 29 PM" src="https://user-images.githubusercontent.com/41960152/131562353-0fe6387a-9219-44b2-9af2-f5e024d6dbf0.png">

<img width="1460" alt="Screen Shot 2021-08-31 at 3 12 52 PM" src="https://user-images.githubusercontent.com/41960152/131562326-7da3f4d8-7910-4b3f-8023-6da6532c3cf8.png">

## Verification Steps

1. update the quay org/user name in the make file and catalog-source.yaml file with your org name
2. `make release`
3. `make catalog-update`
4. make sure the DBaaS operator installed 
5. make sure after successful installation of the DBaaS operator, it installed all its dependencies (CrunchBridge Operator, MongoDB atlas operator, DBaaS Dynamic Plugin<inside >, Telemetry Plugin)



## Type of change


## Checklist (Definition of Done)
